users/models.py

```diff
@@ -53,12 +53,12 @@ class Region(models.Model):
     class Meta:
         db_table = "regions"
 
-class Area(models.Model):
+class SubRegion(models.Model):
     region = models.ForeignKey('Region', on_delete=models.CASCADE)
-    master = models.ForeignKey('Master', on_delete=models.CASCADE)
+    name   = models.CharField(max_length=100)
 
     class Meta:
-        db_table = "areas"
+        db_table = "subregions"
```

Comment:
```suggestion
        db_table = "sub_regions"
```
카테고리 이름이 SubRegion이라면 S와 R을 띄어쓰기로 분리하셨다고 파악하였습니다.
의도와 맞다면 수정하시면 좋을 것 같아요

Reply:
감사합니다! 수정완료했습니다~